rust/src/derivation.rs

```diff
@@ -432,7 +393,7 @@ pub unsafe extern "C" fn zcashlc_derive_address_from_ufvk(
     network_id: u32,
     ufvk: *const c_char,
```

Comment:
Add "The memory referenced by `ufvk` must not be mutated for the duration of the function call."